XamarinCommunityToolkit/Actions/Snackbar/iOS/Extensions/UIViewExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿using UIKit;
```

Comment:
I guess the file name should be UIViewExtensions.ios.cs, right?

Reply:
I added the whole folder to Compile on iOS. Do you think it would be better to rename all files?

Reply:
Ah, yeah, you are right!
No, I think that's fine now